packages/hostScriptsServices/hostScripts/check_docker_network.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+network_name="dncore_network"
+max_retries=5
+retry_delay=60
+
+# Function to check and create network with retry logic
+check_and_create_network() {
+    for ((i=0; i<max_retries; i++)); do
+        if docker network inspect "$network_name" &> /dev/null; then
+            echo "Docker network '$network_name' exists."
+            docker network inspect "$network_name"
```

Comment:
Why do you run it again? Do you want to actually print the output? In that case you might want to store the output and print it after, rather than executing again

Reply:
It might be usefull to know the docker network details, but will remove it